docs/dev-process/pull-request-process.md

```diff
@@ -0,0 +1,51 @@
+# Pull Request Process
+
+Requirements for a pull request to be accepted into PowerShell
+* Writing tests
+* Writing documentation
+
+## Pull Request Workflow
+
+1. The PR *author* creates a pull request from a fork.
+1. The *author* ensures that their pull request passes the [CI system][ci-system] build.
+  - If the build fails, a [Repository Maintainer][repository-maintainer] adds the `Review - waiting on author` label to the pull request. 
+  The *author* can then continue to update the pull request until the build passes.
+3. If the *author* knows whom should participate in the review, they should add them otherwise they can add the recommended *reviewers*.
```

Comment:
this should be `1.` as well to keep consistent with others.

Reply:
I had it as '1', but it doesn't get rendered correctly (at least in VSCode)

Reply:
It should work in Github, see https://github.com/daxian-dbw/PowerShell/blob/temp/docs/dev-process/pull-request-process.md

My quick change is at https://github.com/daxian-dbw/PowerShell/commit/757449e93a00f17a85c3981275da91048d9f5d31. All I did was to change the number to `1.` and make sure there are 3 spaces before every sub-level `-`.

Reply:
Maybe remove "recommended *reviewers*" and say about "code owners" - that we use today.

Reply:
Ok, just an issue with vscode markdown rendering, I'll change all these